### DISCUSSION TITLE
Prevent Artery/OMNeT from Crashing

### DISCUSSION THREAD

**Discussion by HObaid00** [riebl/artery#351] [Q&A] [answered]
Hello everyone, after setting up an artery scenario based on the basic scenariosStoryboard template and inserting more vehicles through SUMO routing, an error comes up:

Wrong type "BaseFrame1609_4" in the @signal[org_car2x_veins_modules_mac_sigRetriesExceeded] property in the "org.car2x.veins.modules.mac.ieee80211p.Mac1609_4" NED type, should be one of: long, unsigned long, double, simtime_t, string, or a registered class name optionally followed by a question mark -- in module (veins::Mac1609_4) World.node[11].nic.mac1609_4 (id=222), at t=11.227182051573s, event #15639

As I am told, the basis of this error might be that Artery no longer sees the vehicle and that it may have exited the simulation either by completing its route or crashing and getting teleported out of the simulation by SUMO. From the base Storyboard and Artery scenario examples, I see that vehicles exiting the simulation should not necessarily be an issue; I have also modified the collision action to none and tried to increase the Playground in case that is the problem, yet the crash persists. If anyone has solved a similar crash with the same "BaseFrame1609_4" error, I would love to know. Currently, I am trying to modify the scenarios/storyboard files and have renamed it to Erlangen based on the default map where I have injected more vehicles which you can find here: https://github.com/3gles00/Cooperative-Behaviour-In-Artery

I really appreciate any help you can provide.
Best regards.

**Comment by signal** [comment]
is it so hard to escape things

**Comment by HObaid00** [comment, marked as answer]
I solved the issue, so the issue arrived when OMNeT could not generate a new node for two incoming vehicles that were too close in space and time, which led to the crash. After generating new trips for the SUMO file and spacing them about 2 s apart, the issue is fixed.  For anyone having similar issues, look if the generated trips are well-spaced in time and space in the OMNeT playground. also, consider if the playground is big enough for your space, although the base 4000x4000 in the storyboard example seems to be big enough for most scenarios.
